bears/c_languages/codeclone_detection/ClangFunctionDifferenceBear.py

```diff
@@ -87,7 +87,7 @@ class ClangFunctionDifferenceBear(GlobalBear):
     REQUIREMENTS = ClangBear.REQUIREMENTS | {PipRequirement('munkres3', '1.0')}
 
     def run(self,
-            counting_conditions: counting_condition_dict=default_cc_dict,
+            counting_conditions: default_cc_dict=counting_condition_dict,
```

Comment:
Can you explain why have you changed this and you have tests failing due to this change ;)

Reply:
So far the only explanation I have is that the default_cc_dict is generated by the counting_condition_dict. So, I made the change locally and the function values didn't appear then.

Reply:
This might need some corresponding changes in the tests as well. Could be the reason for failing tests.